<div align="center">
  <img src="https://github.com/GhedinSGS/.github/blob/main/figures/sgslab_cover.png" alt="Systems Genomics Section" style="width: 100%;">
</div>
<br>

</br>

<h1 align="center">ABOUT US</h1>
<p align="center">The Ghedin Lab is a member of the Laboratory of Parasitic Diseases at the National Institute of Allergy and Infectious Diseases (NIAID) of the NIH in Bethesda, MD. Our research is focused on defining genomic characteristics of human pathogens.  We use a multidisciplinary approach, drawing on genomics, molecular virology, parasitology, and computational biology to unravel the complex relationships between pathogens and their hosts.</p>
<br>

</br>
<h1 align="center">TABLE OF CONTENTS</h1>
<h3 align="center">Select a section below to learn more about our research projects and computational tools:</h3>
<table>
  <tr>
    <td style="text-align: center;">
      <p align="center">Tools & Pipelines</p>
      <a href="https://github.com/GhedinSGS#tools--pipelines">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/Pipelines.png" alt="Pipelines" width="400"/>
      </a>
    </td>
    <td style="text-align: center;">
      <p align = "center">The Microbiome</p>
      <a href="https://github.com/GhedinSGS#the-microbiome">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/Microbiome.png" alt="Microbiome" width="200"/>
      </a>
    </td>
    <td style="text-align: center;">
      <p align="center">Respiratory Virus Evolution & Host Response</p>
      <a href="https://github.com/GhedinSGS#respiratory-virus-evolution--host-response">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/VirusEvolution.png" alt="Respiratory Virus Evolution & Host Response" width="200"/>
      </a>
    </td>
    <td style="text-align: center;">
      <p align="center">Host-Parasite Interactions</p>
      <a href="https://github.com/GhedinSGS#host-parasite-interactions">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/Host-Parasite.png" alt="Host-Parasite" width="200"/>
      </a>
    </td>
  </tr>
</table>
<br>

</br>
<h1 align="center">TOOLS & PIPELINES</h1>
<table>
  <tr>
    <td align="center">
      <a href="https://github.com/GhedinSGS/SGSlab_metagenomics">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/Meta_pipeline.png" alt="Microbiome" width="400" style="margin: auto;"/>
      </a>
      <p>Microbiome Analysis Pipeline</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/DiVRGE">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/divrge.png" alt="DiVRGE" width="175" style="margin: auto;"/>
      </a>
      <p>Deletions in 
        Viral Genomes Extractor</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/timo">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/timo.png" alt="timo" width="150"style="margin: auto;"/>
      </a>
      <p>Single Nucleotide Variant Caller</p>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="">
        <img src="png" alt="CoverCT" width="150" style="margin: auto;"/>
      </a>
      <p>CoverCT</p>
    </td>
    <td align="center">
      <a href="">
        <img src=https://github.com/GhedinSGS/.github/blob/main/figures/mope.png alt="MOPE" width="175" style="margin: auto;"/>
      </a>
      <p>Multi-Omic Pathway Enrichment</p>
    </td>
    <td align="center">
      <a href=https://github.com/GhedinSGS/SARS-CoV-2_analysis">
        <img src="https://github.com/GhedinSGS/.github/blob/main/figures/viruses.png" alt="Viruses" width="150" style="margin: auto;"/>
      </a>
      <p>SARS-CoV-2 Analysis Pipeline</p>
    </td>
  </tr>
</table>
<br>

</br>

<h1 align="center">PROJECTS</h1>
<h4 align="center"> For comprehensive information on project-specific data, analyses, and code, select the links provided below. </h4>
<p></p>
<h2 align="left">The Microbiome</h2>
<h3>Characterizing microbial ecology in upper airways and chronic wounds.</h3>
<p align="center"><i>Our research provides a systems-level landscape of microbial communities, and defines how these communities impact infection outcomes.</i></p>
<table>
  <tr>
    <td align="center">
      <a href="https://github.com/GhedinSGS/SGSlab_metagenomics">
        PROJECT
      </a>
      <p>(CITATION)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/DiVRGE">
        PROJECT
      </a>
      <p>(CITATION)</p>
    </td>
  </tr>
</table>
<br>

</br>
<h2 align="left">Respiratory Virus Evolution & Host Response</h2>
<h3>Investigating drivers and outcomes of respiratory virus evolution.</h3>
<p align="center"><i>We are generating novel insights into the evolutionary dynamics of respiratory pathogens.</i></p>
<table>
  <tr>
    <td align="center">
      <a href="https://github.com/GhedinSGS/SARS-CoV-2_TissueDiversity">
        SARS-CoV-2 Tissue Adaptations
      </a>
      <p>(Johnson KEE et al., 2025)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/hmh_deletions">
        SARS-CoV-2 Deletions
      </a>
      <p><i>(in prep.)</i></p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/HMH_SARS-CoV-2_chronic_infections">
        SARS-CoV-2 in Chronic Infections
      </a>
      <p><i>(in prep.)</i></p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/Virus_Detection_Raman_Wavelength_Comparison">
        Label-free Flu Detection
      </a>
      <p>(Ricker et al., 2024)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/CIVIC_longitudinal_whole_blood_transcriptomics_obesity">
        Flu vaccine response in obese individuals
      </a>
      <p>(Hockman et al., 2025)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/SARS-CoV-2_Antiviral_Resistance">
        SARS-CoV-2 Antiviral Resistance
      </a>
      <p>(Nooruzzaman M*, Johnson KEE*, et al. 2024)</p>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/GhedinSGS/TCD_Mice">
        SARS-CoV-2 in T-cell Deficient Mice 
      </a>
      <p>(Kar et al., 2024)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/SARS-CoV-2-Genomic-Epidemiology-in-the-Dominican-Republic">
        SARS-CoV-2 in the Dominican Republic
      </a>
      <p>(Kreitman A*, Mushegian A*, et al., 2025)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/SARS-CoV-2-genomic-epidemiology-in-Haiti">
        SARS-CoV-2 in Haiti
      </a>
      <p>(Mushegian A*, Kreitman A*, et al.,  2024)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/NA_Vax_Ferret_Transmission">
        Flu Transmission in NA Vaccinated Ferrets
      </a>
      <p>(Septer et al, 2024)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/Flu_DI_study">
        Flu DI222 and Multiciliogenesis
      </a>
      <p>(Wang et al., 2023)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/Optimized-Quantification-of-Intrahost-Viral-Diversity">
        Optimized SNV Detection
      </a>
      <p>(Roder A*, Johnson KEE*, Knoll M* et al., 2023)</p>
    </td>
  </tr>
</table>
<br>

</br>
<h2 align="left">Host-Parasite Interactions</h2>
<h3>Seeking to understand the complex interactions between Brugia malayi (a filarial nematode) and its endosymbiotic bacterium, Wolbachia.</h3>
<p align="center"><i>Our studies address key questions on the reciprocal cues required in the co-dependency between the worm and the bacteria.</i></p>

<table>
  <tr>
    <td align="center">
      <a href="https://github.com/GhedinSGS/Wolbachia-and-ZIKV-replication">
        Wolbachia and ZIKV
      </a>
      <p>(Edwards et al., 2023)</p>
    </td>
    <td align="center">
      <a href="https://github.com/GhedinSGS/Brugia-malayi-microRNAs">
        Brugia malayi miRNAs
      </a>
      <p>(Johnson H. et al., 2024)</p>
    </td>
  </tr>
</table>
<br>

</br>
